source/features/minimize-upload-bar.css

```diff
@@ -0,0 +1,9 @@
+.rgh-has-upload-field .upload-enabled textarea {
+	border-bottom-style: solid;
+	border-bottom-left-radius: 3px;
+	border-bottom-right-radius: 3px;
+}
+
+.rgh-has-upload-field .drag-and-drop {
```

Comment:
Make sure you use the entire selector as it was before because it excluded some other drop areas.

Reply:
I restored https://github.com/sindresorhus/refined-github/blob/43a3ec4e1d9a112f791f58ab2ed868e13b733794/source/features/upload-button.css